CHANGELOG.md

```diff
@@ -18,6 +19,47 @@ CHANGELOG
 
 </details>
 
+Swift 4.1
+---------
+
+* [SE-0185][]
```

Comment:
I think you need to add the link to the actual proposal elsewhere in the file for this to work

Reply:
Much to my surprise, it was already there! (Last line of https://raw.githubusercontent.com/apple/swift/master/CHANGELOG.md.)